packages/connect/src/api/cardano/cardanoCertificate.ts

```diff
@@ -185,12 +210,25 @@ export const transformCertificate = (
         paramsToValidate.push({ name: 'poolParameters', type: 'object', required: true });
     }
 
+    if (
+        certificate.type === PROTO.CardanoCertificateType.STAKE_REGISTRATION_CONWAY ||
+        certificate.type === PROTO.CardanoCertificateType.STAKE_DEREGISTRATION_CONWAY
+    ) {
+        paramsToValidate.push({ name: 'deposit', type: 'uint', required: true });
```

Comment:
Is the stake credential validated somewhere?

Reply:
on line 235